### PR TITLE
Update TensorBoard callback in callbacks.py

### PR DESCRIPTION
**Pull request by 8bitmp3**
See #29958 : Suggested improvements:

* _Added a TensorBoard example:_

Material used and modified from [TensorBoard in Notebooks](https://www.tensorflow.org/tensorboard/r2/tensorboard_in_notebooks) and [Getting started with TensorBoard](https://www.tensorflow.org/tensorboard/r2/get_started).

Note the`%`-sign in front of the command line invocations (for Jupyter).

* _Replaced bullet points `*` with `-` to attempt to fix the way certain bullet points are displayed:_

Previously, they were displayed in one "sentence" as this Markdown `*` syntax wasn't "picked up": `This callback logs events for TensorBoard, including: * Metrics summary plots * Training graph visualization * Activation histograms * Sampled profiling` (For reference, see r1.14 doc [link](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard) and r2.0 doc [link](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/callbacks/TensorBoard)).

### CODE REVIEW

**Comment by 8bitmp3** [comment]
just checking in to see if anything else needs to be done @rchao @gbaned thanks!

File: tensorflow/python/keras/callbacks.py [hunk 1]
```diff
@@ -1378,6 +1378,29 @@ class TensorBoard(Callback):
 
   You can find more information about TensorBoard
   [here](https://www.tensorflow.org/get_started/summaries_and_tensorboard).
+  
+  Example:
+
+  ```python
+  # Import the datetime module to display the date and time
+  import datetime
```
Comment (omalleyt12) [inline tensorflow/python/keras/callbacks.py]: Can you remove the datetime logic? Let's just show the minimum example for this callback

**Review by omalleyt12 — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by gbaned** [comment]
@8bitmp3 Could you please address the reviewer comments. Thanks!

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
